.gitignore

```diff
@@ -2,3 +2,5 @@ token.json
 credentials.json
 .vscode
 budget.egg-info
+config.json
```

Comment:
I reference this file from within `README.md`, and didn't add it into `.gitignore` intentionally so that it could be used as a reference by other people in terms of configuring the app. So it would be great if you added it back 😄